cookbook/vis_mpl_seismic.py

```diff
@@ -0,0 +1,39 @@
+"""
+Seismic: plotting a seismic section from a SEGY
+
+Uses ObsPy package for SEGY file reading
```

Comment:
@eusoubrasileiro final thing, I promise! Could you also add a line here saying something like "you can find instructions for installing Obspy at http://obspy.org/"?